scripts/create-workspace.js

```diff
@@ -287,7 +287,7 @@ async function createWorkspace({
         'test-electron': 'xvfb-maybe electron-mocha --no-sandbox',
       }),
       'test-cov':
-        'nyc -x "**/*.spec.*" --reporter=lcov --reporter=text --reporter=html npm run test',
+        'nyc --compact=false --produce-source-map=false -x "**/*.spec.*" --reporter=lcov --reporter=text --reporter=html npm run test',
```

Comment:
I'll follow up with another patch that will move all the configuration options into a config file, this should be a bit easier to maintain I think. For now I just want to see the CI to pass with minimal amount of changes